README.md

```diff
@@ -58,37 +63,32 @@ windows.
 
 ### Subscribe to events
 
-Subscribing to events is also simple. The following example tells Chasqui to
-forward events to this subscriber's `my-app` queue. Chasqui will detect whether
-you are using Sidekiq or Resque and generate an appropriate worker class for you.
-Within the subscriber block, you use the `on(event_name)` method to register blocks
-of code to handle that event.
+Subscribing to events is also simple. In the following example, we create a
+subscriber to handle events published to the `user.sign-up` channel.
 
     # file: subscriber1.rb
     require 'chasqui'
 
-    Chasqui.subscribe queue: 'my-app' do
+    class UserSignUpSubscriber
+      include Chasqui::Subscriber
+      subscribe channel: 'user.sign-up'
 
-      on 'user.sign-up' do |user_id|
-        # do something when the user signs up
+      def perform(payload)
+        # Do something when the user signs up.
+        #
+        # User.create(name: payload[:user])
```

Comment:
@jbgo should this example say `User.create(name: payload[:name])` instead?  In the snippet above you publish with a payload with `name: 'Darth Vader'`.